src/cli.js

```diff
@@ -35,6 +35,7 @@ Options:
   --v8-cache               Emit a build using the v8 compile cache
   --license [file]         Adds a file containing licensing information to the output
   --stats-out [file]       Emit webpack stats as json to the specified output file
+  --webpack-target         What build target to use for webpack (default: node)
```

Comment:
Lets call this `--target` and accept `es5` so we can always append `node`

Reply:
We can validate the input so that it must begin with `es` like `es5` or `es2020`

Reply:
Updated to accept es values and to always append `node`